### PR TITLE
Add ak8 dqm and fix empty histograms

### PR DESCRIPTION
**Pull request by kirschen**
This reconfigures the JetMET DQM to also monitor AK8 single jet paths and compare them to offline ak8(pfchs) jets.

Corresponds to 
https://its.cern.ch/jira/browse/CMSHLT-826
where a test DQM file is also provided

### CODE REVIEW

File: DQMOffline/Trigger/src/JetMETHLTOfflineSource.cc [hunk 1]
```diff
@@ -456,17 +498,34 @@ JetMETHLTOfflineSource::fillMEforMonAllTrigger(const Event & iEvent, const edm::
...
       
     //L1 and HLT indices
-    edm::InputTag l1Tag(v->getl1Path(),"",processname_);
+    if (verbose_) {
+      cout << "     - L1Path = " << v->getl1Path() << endl;
+      cout << "     - Label  = " << v->getLabel() << endl;
+    }
+
+    //edm::InputTag l1Tag(v->getl1Path(),"",processname_);
+    edm::InputTag l1Tag(v->getLabel(),"",processname_);
```
Comment (kirschen) [inline DQMOffline/Trigger/src/JetMETHLTOfflineSource.cc]: this had to be changed by Isidro to produce non-empty histograms

**Comment by dmitrijus** [comment]
+1

**Comment by kirschen** [comment]
Just realized that this is not merged in, yet...
Could you give a pointer on what needs to be done, please? Or would another test maybe be successful? This fixes empty plots in the JetMET-DQM and is fairly urgent!
--> same for #14706

**Comment by dmitrijus** [comment]
please test

**Comment by kirschen** [comment]
Ok, I see why the test was failing
--> https://github.com/cms-sw/cmssw/pull/14705/files#diff-61419f7cbf758a79b8b4a26ea2f57951R105
expects ak8PFchs-jets to be in the event and the corresponding JEC in the global tag. 
However, the failing test workflows are running with auto:run1_mc as global tag and they only contain ak5/ak7-JEC.
How can this be resolved? I think that setting the input to ak7 just to make the run1-tests pass is not the way to go...?

**Comment by kirschen** [comment]
I ran runTheMatrix.py locally and things seem to work fine. 
As mentioned in the previous comment, some workflows were failing, because offline ak8PFchs-jets +JEC do not seem to be supported by the workflows 9.0 and 23.0. Clearly, the configuration of the module will have to be improved in another PR. However, the included fix of empty histograms is something we need even more urgently in the release.

**Comment by kirschen** [comment]
please test

**Comment by dmitrijus** [comment]
please test

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
+1
